Traceback (most recent call last):
  File "/mnt/netapp2/Home_FT2/home/ulc/cursos/curso342/bolmarauto/scripts/sbatch/../sk_learn/TFM2/MejoresKNeighbors.py", line 90, in <module>
    grid_search.fit(input, y)
  File "/mnt/netapp2/Store_uni/home/ulc/cursos/curso342/miniconda3/envs/genhpc/lib/python3.12/site-packages/sklearn/base.py", line 1389, in wrapper
    return fit_method(estimator, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/netapp2/Store_uni/home/ulc/cursos/curso342/miniconda3/envs/genhpc/lib/python3.12/site-packages/sklearn/model_selection/_search.py", line 1024, in fit
    self._run_search(evaluate_candidates)
  File "/mnt/netapp2/Store_uni/home/ulc/cursos/curso342/miniconda3/envs/genhpc/lib/python3.12/site-packages/sklearn/model_selection/_search.py", line 1571, in _run_search
    evaluate_candidates(ParameterGrid(self.param_grid))
  File "/mnt/netapp2/Store_uni/home/ulc/cursos/curso342/miniconda3/envs/genhpc/lib/python3.12/site-packages/sklearn/model_selection/_search.py", line 970, in evaluate_candidates
    out = parallel(
          ^^^^^^^^^
  File "/mnt/netapp2/Store_uni/home/ulc/cursos/curso342/miniconda3/envs/genhpc/lib/python3.12/site-packages/sklearn/utils/parallel.py", line 77, in __call__
    return super().__call__(iterable_with_config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/netapp2/Store_uni/home/ulc/cursos/curso342/miniconda3/envs/genhpc/lib/python3.12/site-packages/joblib/parallel.py", line 1952, in __call__
    return output if self.return_generator else list(output)
                                                ^^^^^^^^^^^^
  File "/mnt/netapp2/Store_uni/home/ulc/cursos/curso342/miniconda3/envs/genhpc/lib/python3.12/site-packages/joblib/parallel.py", line 1595, in _get_outputs
    yield from self._retrieve()
  File "/mnt/netapp2/Store_uni/home/ulc/cursos/curso342/miniconda3/envs/genhpc/lib/python3.12/site-packages/joblib/parallel.py", line 1699, in _retrieve
    self._raise_error_fast()
  File "/mnt/netapp2/Store_uni/home/ulc/cursos/curso342/miniconda3/envs/genhpc/lib/python3.12/site-packages/joblib/parallel.py", line 1734, in _raise_error_fast
    error_job.get_result(self.timeout)
  File "/mnt/netapp2/Store_uni/home/ulc/cursos/curso342/miniconda3/envs/genhpc/lib/python3.12/site-packages/joblib/parallel.py", line 736, in get_result
    return self._return_or_raise()
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/netapp2/Store_uni/home/ulc/cursos/curso342/miniconda3/envs/genhpc/lib/python3.12/site-packages/joblib/parallel.py", line 754, in _return_or_raise
    raise self._result
joblib.externals.loky.process_executor.TerminatedWorkerError: A worker process managed by the executor was unexpectedly terminated. This could be caused by a segmentation fault while calling the function or by an excessive memory usage causing the Operating System to kill the worker.

The exit codes of the workers are {SIGKILL(-9)}
slurmstepd: error: Detected 1 oom-kill event(s) in StepId=14380851.batch. Some of your processes may have been killed by the cgroup out-of-memory handler.
